velox/exec/HashTable.h

```diff
@@ -639,11 +655,8 @@ class HashTable : public BaseHashTable {
   uint8_t* FOLLY_NULLABLE tags_ = nullptr;
   char* FOLLY_NULLABLE* FOLLY_NULLABLE table_ = nullptr;
   memory::MappedMemory::ContiguousAllocation tableAllocation_;
-  int64_t size_ = 0;
+  HashTableStats stats_;
```

Comment:
Those counters are not for stats purpose only. So keep those counters and generate HashTable::Stats from those counters in s
stats() call.
```
s/HashTableStats/HashTable::Stats/
```